### PR TITLE
Additions to test_netstats and removal of redundancy in netstats

### PR DESCRIPTION
**Pull request by alexayala08**
Added tests for get_prop_type() and nx2gt().

Removed redundant code in pynets/stats/netstats

### CODE REVIEW

File: tests/test_netstats.py [hunk 1]
```diff
@@ -16,6 +16,31 @@
...
+@pytest.mark.parametrize("value", [True, 3.14, "word", {"project":"PyNets"}])
+def test_get_prop_type(value):
+    """
+    Test for get_prop_type() functionality
+    """
+    start_time = time.time()
+    tname, value, key = netstats.get_prop_type(value)
+    print("%s%s%s" % ('thresh_and_fit (Functional, proportional thresholding) --> finished: ',
+                      np.round(time.time() - start_time, 1), 's'))
+    assert tname is not None
```
Comment (dPys) [inline tests/test_netstats.py:28]: Can form more meaningful assert statement here given that we know tname will have to take on one of either: 
'bool', 'float', 'string', 'object'.

File: tests/test_netstats.py [hunk 2]
```diff
@@ -16,6 +16,31 @@
...
+    """
+    base_dir = str(Path(__file__).parent/"examples")
+    in_mat = np.load(f"{base_dir}/miscellaneous/graphs/002_modality-func_rsn-Default_model-cov_nodetype-spheres-2mm_smooth-2fwhm_hpass-0.1Hz_thrtype-PROP_thr-0.95.npy")
+    nxG = nx.from_numpy_array(in_mat)
+
+    start_time = time.time()
+    gtG = netstats.nx2gt(nxG)
+    print("%s%s%s" % ('thresh_and_fit (Functional, proportional thresholding) --> finished: ',
+                      np.round(time.time() - start_time, 1), 's'))
+    assert gtG is not None
```
Comment (dPys) [inline tests/test_netstats.py:42]: Here too, we should expand upon this assert. One thing to consider is that if nxG is not None (i.e. and it's a true nx.Graph versus, say, a np.ndarray), then gtG should always return a valid graph_tool graph object.

**Review by dPys — CHANGES_REQUESTED** [review]
@alexayala08 -- nice work on this. Just a couple comments/suggestions to address before we go ahead and merge. Before doing anything new though, go ahead and run `git fetch upstream` and `git merge origin/development` since I've merged some changes from earlier that are not yet synced with the dev branch on your fork.

**Comment by alexayala08** [comment]
The failed tests seem to because I am importing graph_tool. Should I import graph_tool in the individual test functions similar to how it is done in tests/netstats where it is inside of a try and except statement?

**Comment by dPys** [comment]
> The failed tests seem to because I am importing graph_tool. Should I import graph_tool in the individual test functions similar to how it is done in tests/netstats where it is inside of a try and except statement?

Yes! That's exactly the way to do it :-)

**Review by dPys — APPROVED** [review]

**Comment by dPys** [comment]
Merging with development! :-)

**Pull request merged**